webpack/assets/javascripts/react_app/common/variables.scss

```diff
@@ -1,3 +1,3 @@
-@import '~@theforeman/vendor/scss/variables';
+@import 'node_modules/@theforeman/vendor/scss/variables.scss';
```

Comment:
Is it normal to include `node_modules` in a path?

Reply:
jenkins doesnt understand `~@theforeman` for some reasons and I'm trying to see whats going on there (which is also why there are so many pushes today).
Locally this works fine

Reply:
Sounds like something that needs further investigation. I bet it's something subtle in some version difference of some library or something being present/absent.